### ISSUE TITLE
Dates from Jan 1st to 20th don't generate chart

### ISSUE THREAD

**Issue by hjimenez** [0xStarcat/CircularNatalHoroscopeJS#14]
Hi,

I tried to generate a chart for any location on any year using days from January 1st to the 20th and nothing gets generated.

Can you please confirm if this is an error?

H

**Comment by KevinMartins367** [comment]
did you use Origin with all the information? Like:
```
const origin = new Origin ({
   year: 2021,
   month: 0
   date: 20,
   hour: 16,
   minute: 30,
   latitude: 40.0,
   longitude: -70.0,
});
```

**Comment by hjimenez** [comment]
Yes.
If I open up the browser console I get a 
`this.sunSignElement.innerHTML = horoscope.SunSign.label;` undefined error, so seems like the Capricorn sign label is not showing for some reason.
 I am not 100% javascript fluent, trying to see where, but I put console.log() into parts of the script and I do see the Origin object fully populated.

**Comment by 0xStarcat** [comment]
I'll have a look soon! thanks for reporting

**Comment by 0xStarcat** [comment]
I was able to reproduce this in the demo - you're correct SunSign is returning undefined from Jan 1 - Jan 20th. I'm looking into debugging it now

**Comment by 0xStarcat** [comment]
resolved with #19  and released with https://github.com/0xStarcat/CircularNatalHoroscopeJS/releases/tag/v1.0.14
